src/app/components/content-action-bar/ContentActionBar.test.js

```diff
@@ -0,0 +1,75 @@
+import React from "react";
+import ContentActionBar from "./ContentActionBar.jsx";
+import renderer from "react-test-renderer";
+import { mount } from "enzyme";
+
+describe("ContentActionBar", () => {
+    describe("is created stuck at bottom with a single action button, cancel and with unsaved changes", () => {
+        const contentActionBarProps = {
+            buttons: [
+                {
+                    id: "foo-id",
+                    text: "foo",
+                    interactionCallback: () => {},
+                    style: "positive",
+                    disabled: false
+                }
+            ],
+            cancelCallback: () => {},
+            stickToBottom: true,
+            unsavedChanges: true
+        };
+
+        const component = mount(<ContentActionBar {...contentActionBarProps} />);
+        it("should display 2 buttons, be attached at the bottom and have a warning that there are unsaved changes", () => {
+            expect(component.find(".content-action-bar--bottom-fixed").length).toBe(1);
+            expect(component.find(".btn").length).toBe(2);
+            expect(component.find("#btn-cancel").exists()).toBe(true);
+            const fooButton = component.find("#foo-id");
+            expect(fooButton.exists()).toBe(true);
+            expect(fooButton.hasClass("btn--positive")).toEqual(true);
+            expect(component.find(".content-action-bar__warn").exists()).toBe(true);
+        });
+        const componentForSnapshot = renderer.create(<ContentActionBar {...contentActionBarProps} />);
+        expect(componentForSnapshot.toJSON()).toMatchSnapshot();
```

Comment:
can you please extract this to separate test case. 
it("matches the snapshot", () => {

}